### ISSUE TITLE
[BUG] Regional Price Comparison not displaying correctly

### ISSUE THREAD

**Issue by AKAdb-DavieJones** [jshackles/Enhanced_Steam#1501]
Hi, I live in Argentina, and local currency (ARS) has been recently enabled for us on the Steam Store.

Ever since the change, I've noticed that the globe indicator has disappeared, and nothing pops up when I move my mouse over the price. Here are my usual settings:
![image](https://user-images.githubusercontent.com/34213012/33526452-a3c91316-d820-11e7-9b8b-d1ddb972db13.png)

I've also noticed that, if I change the setting to have the prices shown in USD (
![image](https://user-images.githubusercontent.com/34213012/33526461-d732b964-d820-11e7-8440-d9f267853920.png)), then **prices are still shown in ARS**, but at least the globe indicator returns.
Sadly, when I mouse over it, there is still an issue with the percentages comparison:
![image](https://user-images.githubusercontent.com/34213012/33526487-238eab1a-d821-11e7-950d-542ebe9a038c.png)

I want to end by thanking you for all the amazing work you do on this extension. You are awesome!!

Cheers,
AKA

**Comment by santidaross** [comment]
Yes, this is an issue with the new currencies and jshackles is [working on an update.](http://steamcommunity.com/groups/enhancedsteam/discussions/0/1486613649676112442/#c1486613649677735291)

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
